api/conftest.py

```diff
@@ -120,6 +121,31 @@ def django_db_setup(request: pytest.FixtureRequest) -> None:
         db_settings["NAME"] = test_db_name
 
 
+@pytest.fixture(autouse=True)
+def restrict_http_requests(monkeypatch: pytest.MonkeyPatch) -> None:
```

Comment:
I'd love to see a docstring here explaining this fixture.

Reply:
Done.

Reply:
Done.